airflow/contrib/operators/dataproc_operator.py

```diff
@@ -377,6 +377,132 @@ def execute(self, context):
         self._wait_for_done(service)
 
 
+class DataprocClusterScaleOperator(BaseOperator):
+    """
+    Scale a cluster on Google Cloud Dataproc. The operator will wait until the
+    cluster is re-scaled.
+
+    :param cluster_name: The name of the cluster to create.
+    :type cluster_name: string
+    :param project_id: The ID of the google cloud project in which
+        the cluster runs
+    :type project_id: string
+    :param region: leave as 'global', might become relevant in the future
```

Comment:
Can you update this to `The region for the dataproc cluster` as now it does support regions like `europe-west`